src/SwitchSelector/SwitchSelector.tsx

```diff
@@ -0,0 +1,68 @@
+import clsx from "clsx";
+import React from "react";
+import { Button, makeStyles } from "..";
+
+type SwitchSelectorOptions = {
+  label: string | React.ReactNode;
+  value: string;
+};
+
+interface SwitchSelectorProps {
+  options: SwitchSelectorOptions[];
+  setActiveTab: (index: string) => void;
+  activeTab?: string;
+}
+
+const useStyles = makeStyles((theme) => ({
+  switchSelector: {
+    border: `1px solid ${theme.palette.saleor.main[5]}`,
+    padding: "6px",
+    borderRadius: 4,
+    width: "fit-content",
+    "&:not(:first-child)": {
+      marginLeft: "6px",
+    },
+  },
+  button: {
+    color: "#77738C",
+    background: "none",
+    borderColor: "transparent",
+    cursor: "pointer",
+
+    "&:hover": {
+      backgroundColor: theme.palette.common.white,
+      color: theme.palette.saleor.active[1],
+      border: `1px solid ${theme.palette.saleor.active[1]}`,
+    },
+  },
+  buttonActive: {
+    color: theme.palette.common.white,
+    backgroundColor: theme.palette.saleor.active[1],
+  },
+}));
+
+export const SwitchSelector: React.FC<SwitchSelectorProps> = ({
+  options,
+  setActiveTab,
+  activeTab,
+}) => {
+  const classes = useStyles();
+
+  return (
```

Comment:
If it supports more than two options, then i think its better to go for compound component pattern. Mapping children led us in dashboard to constant need for extending choice interface.

Reply:
So I should make two components, one that is a wrapper eg. `SwitchSelector` and the buttons eg. `SwitchSelectorButton` in macaw, then in dashboard do something like:

```
<SwitchSelector>
   <SwitchSelectorButton name={optionOneName} onClick={onClick}>{optionOneLabel}</SwitchSelectorButton>
   <SwitchSelectorButton name={optionTwoName} onClick={onClick}>{optionTwoLabel}</SwitchSelectorButton>
   <SwitchSelectorButton name={optionThreeName} onClick={onClick}>{optionThreeLabel}</SwitchSelectorButton>
   ...
</SwitchSelector>
```
?